src/blocks/discord/discord-canvas/rankcard/set_badge.js

```diff
@@ -38,7 +38,7 @@ Blockly.Blocks[blockName] = {
 };
 
 Blockly.JavaScript[blockName] = function(block){
-    const number = Blockly.JavaScript.valueToCode(block, "SLIDER", Blockly.JavaScript.ORDER_ATOMIC) || null;
+    const number = Blockly.JavaScript.valueToCode(block, "SLIDER", Blockly.JavaScript.ORDER_ATOMIC) || blockData.args0[1].value;
```

Comment:
github  is refusing to register the change in the review so to be clear this is what im talking abt